letsencrypt/plugins/manual.py

```diff
@@ -26,13 +26,18 @@
 class Authenticator(common.Plugin):
     """Manual Authenticator.
 
-    .. todo:: Support for `~.challenges.DVSNI`.
+       This plugin requires user's manual intervention in setting up a HTTP
+       server for solving SimpleHTTP challenges and thus does not need to be
+       run as a privilidged process. Alternatively shows instructions on how
+       to use Python's built-in HTTP server.
 
+    .. todo:: Support for `~.challenges.DVSNI`.
     """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
+    hidden = True
 
-    description = "Manual Authenticator"
+    description = "Manually configure and run a simple Python webserver"
```

Comment:
And this won't be even displayed in the UI, because the plugin is hidden. Which means that only technically inclined people will read it (those that go into the source code). They will be confused by this statement.